scripts/build-performance-tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Can the scripts go back into `dev`, please? 

Originally we had both, but scripts ended up in `dev` and ultimately they're only used by the developers of the project anyway so `dev` felt more appropriate. The other benefit to just having `dev` is that it's a little tidier at the top-level.